src/main/kotlin/org/partiql/scribe/sql/SqlLayout.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+package org.partiql.scribe.sql
+
+/**
+ * [SqlLayout] determines how an [SqlBlock] tree is transformed in SQL text.
+ */
+public interface SqlLayout {
+
+    public fun format(head: SqlBlock): String
+
+    public companion object {
+
+        /**
+         * Default SQL format.
+         */
+        @JvmStatic
+        public val DEFAULT = object : SqlLayout {
+
+            private val indent = Indent(2, Indent.Type.SPACE)
```

Comment:
if we're making the `indent` non-configurable for the `SqlLayout` this value and the `Indent` class definition on L99 could be deleted?